components/automate-ui/src/app/pages/+compliance/shared/reporting/report-query.service.ts

```diff
@@ -31,7 +31,7 @@ export class ReportQueryService {
 
   addFilter(filter) {
     const filters = this.filters.getValue();
-    if (filter.type.name === 'profile') {
+    if (filter.type.name === 'profile' && filter.value.version !== '') {
```

Comment:
Only add the version as part of the name if there is a version. There is not a version if it is a wildcard filter.